src/adapters/node-http/core.ts

```diff
@@ -42,12 +41,8 @@ export const createOpenApiNodeHttpHandler = <
 ) => {
   const router = cloneDeep(opts.router);
 
-  // Validate router
-  generateOpenApiDocument(router, { title: '-', version: '-', baseUrl: '-' });
-  monkeyPatchVoidInputs(router);
```

Comment:
Same issue as above. It's better to monkey patch when needed instead of monkey patching the full router upfront.